src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -362,7 +369,7 @@ cbuf_referenced(struct cbuf_info *cbi)
 	struct cbuf_meta *mt, *own_mt = m->m;
 
 	old_nfo = own_mt->nfo;
-	new_nfo = old_nfo | CBUF_INCONSISENT;
+	new_nfo = old_nfo | CBUF_INCONSISTENT;
```

Comment:
Yeah, I realized the same thing but only AFTER I had made the changes over winter break.